ramls/invoice.raml

```diff
@@ -105,3 +108,33 @@ resourceTypes:
     get:
       description: Get system generated Invoice Number
       is: [validate]
+  /acquisitions-unit-assignments :
+    displayName: Acquisitions unit assignments
+    description: Create acquisitions unit assignments record
+    type:
+      collection:
```

Comment:
`collection.raml` recommends to include [`orderable.raml` ](https://github.com/folio-org/raml/blob/1747d2077e21a2662ff6a59b71cf15ec10d090bf/rtypes/collection.raml#L5) but we are not including?

Reply:
The acquisitionsUnitAssignments contains only UUIDs. I don't think anyone would want to sort by UUID.